src/vmm/src/builder.rs

```diff
@@ -531,18 +526,12 @@ pub fn build_microvm_from_snapshot(
     params: &LoadSnapshotParams,
     vm_resources: &mut VmResources,
 ) -> Result<Arc<Mutex<Vmm>>, BuildMicrovmFromSnapshotError> {
-    // TODO: take it from kvm-bindings when userfault support is merged upstream
-    const KVM_CAP_USERFAULT: u32 = 241;
```

Comment:
do we not check for it anymore?

Reply:
Ah, I think I originally forgot to update this and then noticed quite late into testing, so I just dropped the check instead of updating it with the correct number. But should readd this, thanks.